default.yml

```diff
@@ -1,12 +1,17 @@
 require:
 - rubocop-rails
 - rubocop-rspec
+- ./custom_cops/nxt_core/rails/use_of_rails_logger.rb
+- ./custom_cops/nxt_core/rails/missing_logger_include.rb
 
 inherit_mode:
   merge:
     - Exclude
     - Include
 
+Rails/MissingLoggerInclude:
```

Comment:
Can you please add a comment that this is custom cop? Or maybe change the name to make it obvious 

E.g. `NxtCop/MissingLoggerInclude`

Reply:
Found a sweet spot. Using the `Description` field.